docs/samples/Microsoft.ML.Samples/Dynamic/ProjectionTransforms.cs

```diff
@@ -37,7 +37,7 @@ public static void Example()
             };
 
             // A pipeline to project Features column into Random fourier space.
-            var rffPipeline = ml.Transforms.Projection.CreateRandomFourierFeatures(nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), newDim: 4);
+            var rffPipeline = ml.Transforms.RandomFourierExpand(nameof(SamplesUtils.DatasetUtils.SampleVectorOfNumbersData.Features), dimension: 4);
```

Comment:
>Expand [](start = 57, length = 6)

Expand assumes you can only grow in size.
Maybe just `RandomFourierDecomposition`?

Reply:
Your argument is not correct. This is a process for approximating (infinite) number of features.

Reply:
[https://www.merriam-webster.com/dictionary/expand](https://www.merriam-webster.com/dictionary/expand)
1: to open up : UNFOLD
2 : to increase the extent, number, volume, or scope of : ENLARGE
3a : to express at length or in greater detail
bla bla bla.

In this example you take vector of 10 features and map it into 4 dimensions.
How this is expand? It's a RandomFourierShrink!
I now you can put bigger number, but would be nice to have verb which serve both cases, rather than one of them.
RandomFourierMapping maybe?

---
In reply to: [263594873](https://github.com/dotnet/machinelearning/pull/2865#discussion_r263594873) [](ancestors = 263594873)

Reply:
Ok. I will do
```csharp
        public static RandomFourierKernelMappingEstimator RandomFourierKernelMap(this TransformsCatalog catalog,

```

---
In reply to: [263609876](https://github.com/dotnet/machinelearning/pull/2865#discussion_r263609876) [](ancestors = 263609876,263594873)